src/test/java/ca/uhn/fhir/jpa/starter/MultitenantServerR4IT.java

```diff
@@ -103,5 +105,10 @@ void beforeEach() {
     ourClient = ourCtx.newRestfulGenericClient(ourServerBase);
     ourClient.registerInterceptor(new LoggingInterceptor(true));
     ourClient.registerInterceptor(ourClientTenantInterceptor);
+ // Create an HTTP basic auth interceptor
+ 	String username = "hapi";
+ 	String password = "hapi123";
+ 	IClientInterceptor authInterceptor = new BasicAuthInterceptor(username, password);
+ 	ourClient.registerInterceptor(authInterceptor);
```

Comment:
@gitcliff , revert these changes ..these are just example servers ..not the actual running servers